src/components/Form/Form.jsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
Can you add a screenshot and description of the error on this place, please?

Reply:
![image](https://user-images.githubusercontent.com/61360270/142057753-878240fc-b01b-4730-85d2-7271d94422e4.png)

Reply:
This is because the propType is not set correctly.

@Katsu08 please help me to add a ticket about changing the way we handle Form and FormInputs, in this new Form we can solve this and other problems current implementation have